external/kafka-0-10/src/main/scala/org/apache/spark/streaming/kafka010/KafkaRDD.scala

```diff
@@ -64,16 +64,12 @@ private[spark] class KafkaRDD[K, V](
       " must be set to false for executor kafka params, else offsets may commit before processing")
 
   // TODO is it necessary to have separate configs for initial poll time vs ongoing poll time?
-  private val pollTimeout = conf.getLong("spark.streaming.kafka.consumer.poll.ms",
+  private val pollTimeout = conf.get(CONSUMER_POLL_MS).getOrElse(
     conf.getTimeAsSeconds("spark.network.timeout", "120s") * 1000L)
```

Comment:
Can we use the existing config in core for this?

Reply:
> Can we use the existing config in core for this?

What do you mean by replacing `spark.network.timeout` with existing config in core?  I checked the code in core have config `spark.network.timeout` not yet.

Reply:
See `.../internal/config/Network.scala` in core

Reply:
> See `.../internal/config/Network.scala` in core

OK. I didn't notice this class.

Reply:
> See `.../internal/config/Network.scala` in core

Thanks, I have replace this config.